docs/output.md

```diff
@@ -357,6 +366,65 @@ Output Summaries:
 
 [RGI](https://github.com/arpcard/rgi) (**R**esistance **G**ene **I**dentifier) predicts resistome(s) from protein or nucleotide data based on homology and SNP models. It uses reference data from the Comprehensive Antibiotic Resistance Database (CARD).
 
+#### argNorm
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `normalized/`
+  - `*.{tsv}`: search results in tabular format
+  </details>
+  <details markdown="1">
+  <summary>ARG summary table headers</summary>
+
+| Table column                                      | Description                                                                                                                                                                         |
```

Comment:
I don't think it makes sense for this massive table to be here.

Either it should goin the collapseable section, or even better: should go on the `argNorm` documentation (which this very useful bit of information doesn't appear to be there!) and we link there!

Reply:
In the current state, the table is reduced to 3 columns. I'd rather go for all columns collapsable or link to argNorm docs as James said. Why did you retain 3 columns? A link will be easier to maintain, although I personally find a collapsable column explanation nicer. No strong opinion from my side. If you go for a link, please add a sentence like "Details on the output table can be found in the [argNorm documentation](https://github.com/BigDataBiology/argNorm/blob/main/README.md)."

Reply:
My idea was to show the columns that argNorm would add to the hamronized output. But looking at it now, I think I should make it clear that the 3 columns are added to the columns in hamronization, providing a link to the collapsable column explanation for hamronization. Would that be ok?

Reply:
Ahhh yes! That caused my confusion and it would be great to add that info. I just compared argnorm table to the hamronization table and it is actually the same + 3 columns, sorry I didn't get that before :smile: